core/src/main/scala/org/apache/pekko/persistence/r2dbc/internal/MonoToFuture.scala

```diff
@@ -0,0 +1,59 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * license agreements; and to You under the Apache License, version 2.0:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
```

Comment:
is it possible that we should be using the standard ASF header here? this Akka derived header is only for Akka derived code

Reply:
Will update later